### PR TITLE
Update semanal_enum.py

### PR DESCRIPTION
**Pull request by Arshaan-256**
Relaxed the checks for IntEnum arguments.

### CODE REVIEW

**Comment by Michael0x2a** [comment]
Thanks for the PR!

It looks like some of the tests are broken -- can you fix them and add a few more? You can find more information on how to run and write tests at https://github.com/python/mypy#tests.

File: mypy/semanal_enum.py [hunk 1]
```diff
@@ -107,7 +107,8 @@ def parse_enum_call_args(self, call: CallExpr,
         if len(args) < 2:
             return self.fail_enum_call_arg("Too few arguments for %s()" % class_name, call)
         if len(args) > 2:
-            return self.fail_enum_call_arg("Too many arguments for %s()" % class_name, call)
+            if call.arg_kinds != [ARG_POS, ARG_POS, ARG_POS]:
```
Comment (JelleZijlstra) [inline mypy/semanal_enum.py]: This doesn't make much sense in the light of the next `if` condition: if this one passes the next one (on line 112) will still fail.

Comment (Arshaan-256) [reply mypy/semanal_enum.py]: @JelleZijlstra Thanks for your review. Could you tell me what does `ARG_POS` mean?

File: mypy/semanal_enum.py [hunk 2]
```diff
@@ -107,9 +107,11 @@ def parse_enum_call_args(self, call: CallExpr,
         if len(args) < 2:
             return self.fail_enum_call_arg("Too few arguments for %s()" % class_name, call)
         if len(args) > 2:
-            return self.fail_enum_call_arg("Too many arguments for %s()" % class_name, call)
-        if call.arg_kinds != [ARG_POS, ARG_POS]:
-            return self.fail_enum_call_arg("Unexpected arguments to %s()" % class_name, call)
+            if call.arg_kinds != [ARG_POS, ARG_POS, ARG_NAMED_OPT]:
```
Comment (Michael0x2a) [inline mypy/semanal_enum.py]: Enum can accept up to [four named arguments](https://docs.python.org/3/library/enum.html#functional-api), not just one.

You should also try checking `call.arg_names` to make sure the arguments are what we expect.

Comment (Arshaan-256) [reply mypy/semanal_enum.py]: @Michael0x2a Sure! Will do it.

File: mypy/checkexpr.py [hunk 3]
```diff
@@ -3424,6 +3424,8 @@ def visit_super_expr(self, e: SuperExpr) -> Type:
 
         # We have an expression like super(T, var).member
 
+        print(SuperExpr)
```
Comment (ethanhs) [inline mypy/checkexpr.py]: I think you left this debug print in on accident.
```suggestion

```

**Pull request closed without merging**